spec/controllers/annotations_controller_spec.rb

```diff
@@ -448,7 +452,7 @@
         expect(AnnotationTagMap.all).to_not eq(tag_map_count)
       end
 
-      it 'detelets the list_annotations map' do
+      it 'deletes the list_annotations map' do
         # the list it belonged to no longer has this annotation ID
         annotation_id = Annotation.last.annotation_id
         delete :destroy, format: :json, id: annotation_id
```

Comment:
Although not changed, similar idea to the put command in terms of updating to one function or not.